src/components/HomeSection.jsx

```diff
@@ -5,61 +5,62 @@ import { Container, Grid, Typography } from '@mui/material';
 const HomeSection = ({
   alignItems,
   direction,
-  tab,
   title,
+  main,
   description,
   buttons,
   pic,
-}) => (
-  <Container sx={{ padding: 4 }}>
-    <Grid container spacing={2} direction={direction} alignItems={alignItems}>
-      <Grid item xs={12} sm={6}>
-        <Typography variant="overline" component="h6" gutterBottom>
-          {tab}
-        </Typography>
-        <Typography variant="h2" component="h2" gutterBottom>
-          {title}
-        </Typography>
-        <Typography variant="body2" paragraph>
-          {description}
-        </Typography>
-        <Grid container spacing={2}>
-          {buttons.map((button, i) => (
-            // eslint-disable-next-line react/no-array-index-key
-            <Grid item key={i}>
-              {button}
-            </Grid>
-          ))}
+}) => {
+  const picSize = main ? '90%' : '80%';
+  return (
+    <Container sx={{ padding: 4 }}>
+      <Grid container spacing={3} direction={direction} alignItems={alignItems}>
+        <Grid item xs={12} sm={6}>
+          <Typography variant={main ? 'h2' : 'h3'} component="h2" gutterBottom>
+            {title}
+          </Typography>
+          <Typography variant="body1" paragraph>
+            {description}
+          </Typography>
+          <Grid container spacing={2}>
+            {buttons.map((button, i) => (
+              // eslint-disable-next-line react/no-array-index-key
+              <Grid item key={i}>
+                {button}
+              </Grid>
+            ))}
+          </Grid>
+        </Grid>
+        <Grid item xs={12} sm={6}>
+          <img
+            src={pic}
+            alt=""
+            style={{
+              width: picSize,
+              height: picSize,
```

Comment:
One thing I'm noticing is that the height of each tab is slightly different

Reply:
It's actually collapsed when a picture is loading.

Reply:
fixed by setting size of grid item